### PR TITLE
CCM-SMP support - updated the what's supported section 

### PR DESCRIPTION
**Pull request by charanya-j**
Preview link: https://64f5ef6adbe81d5a10f3c48f--harness-developer.netlify.app/docs/cloud-cost-management/whats-supported

### CODE REVIEW

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md [hunk 1]
```diff
@@ -575,9 +575,29 @@ Harness CCM Budgets allow you to set custom budgets and receive alerts when your
...
 ```
-
-
 :::important note
 Harness does not currently support AWS China regions.
 :::
+
+## CCM on Harness Self-Managed Enterprise Edition (SMP)
+
+The following table provides the feature support matrix for CCM on Harness SMP. For more information, go to [Self-Managed Enterprise Edition](../../self-managed-enterprise-edition.md) and [CCM on SMP](../2-getting-started-ccm/ccm-smp/1-smp-ccm-roadmap.md).
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/shared/ccm-supported-platforms.md:584]: We should also add a note on the support for the infra where the SMP install happens. We only support AWS infra installs for CCM.

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md [hunk 2]
```diff
@@ -575,9 +575,29 @@ Harness CCM Budgets allow you to set custom budgets and receive alerts when your
...
+  
+| **Features** | **AWS** | **Azure** | **GCP** | **Kubernetes** | **Connected environment** | **Air-gapped environment** |
+| --- | --- | --- | --- | --- | --- | --- |
+|Perspectives | ✅  | ❌ | ❌ | ✅ | ✅ | ✅ |
+|Cost categories | ✅  | ❌ | ❌ | ✅ | ✅ | ✅ |
+|Budgets | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ | ✅ |
+|BI dashboards |✅  | ❌ | ❌ | ✅ | ✅ | ✅ |
+|Anomaly detection  | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ |
+|Currency standardization | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+|Recommendations | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/shared/ccm-supported-platforms.md:594]: We don't support AWS Recommendations.

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md [hunk 3]
```diff
@@ -575,9 +575,29 @@ Harness CCM Budgets allow you to set custom budgets and receive alerts when your
...
+|AutoStopping rules | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/shared/ccm-supported-platforms.md:595]: Let's change this to AutoStopping

**Review by Rohit-Kaliki — CHANGES_REQUESTED** [review]

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md (hunk 1 above)
Comment (KimberlyFields) [reply docs/cloud-cost-management/shared/ccm-supported-platforms.md]: Done

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md (hunk 3 above)
Comment (KimberlyFields) [reply docs/cloud-cost-management/shared/ccm-supported-platforms.md]: Done

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md (hunk 2 above)
Comment (KimberlyFields) [reply docs/cloud-cost-management/shared/ccm-supported-platforms.md]: Fixed

**Comment by KimberlyFields** [comment]
@Rohit-Kaliki I made your suggested changes. Please review and approve if it looks good now.

**Review by Rohit-Kaliki — APPROVED** [review]

**Comment by Rohit-Kaliki** [comment]
@KimberlyFields made one change, supported infra was AWS not Azure

**Pull request merged**